tests/pages/adminPage.js

```diff
@@ -175,4 +177,35 @@ module.exports = {
     I.pressKey(['Control', 'a']);
     I.pressKey('Backspace');
   },
+
+  /**
+   * Encapsulates Tooltip data verification.
+   * There could be only one tooltip popup on a page.
+   *
+   * @param   tooltipObj        one of {@link pmmSettingsPage.tooltips} or an object with similar structure
+   * tooltipObj have to have field tooltipText containing selector for tooltip text and field tooltipReadMoreLink
+   * containing selector for tooltip doc link
+   * @returns {Promise<void>}   requires await in test body.
+   */
+  async verifyTooltip(tooltipObj) {
+    const tooltipIcon = tooltipObj.iconLocator;
+    const tooltipText = tooltipObj.tooltipText.as('Tooltip text');
+
+    I.waitForVisible(tooltipIcon, 5);
+    I.moveCursorTo(tooltipIcon);
+    I.scrollTo(tooltipText);
+    I.waitForVisible(tooltipText, 5);
+    I.seeTextEquals(tooltipObj.text, tooltipText);
+    /* there are tooltip without "Read more" link */
+    if (tooltipObj.link) {
+      const tooltipReadMoreLink = tooltipObj.tooltipReadMoreLink.as(`Tooltip "Read more" link for ${tooltipObj.iconLocator}`);
+
+      I.scrollTo(tooltipReadMoreLink);
+      I.seeAttributesOnElements(tooltipReadMoreLink, { href: tooltipObj.link });
+      const readMoreLink = (await I.grabAttributeFrom(tooltipReadMoreLink, 'href'));
+      const response = await I.sendGetRequest('https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types1233333333.html');
```

Comment:
I think you forgot to replace the "https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types1233333333.html" here?

Reply:
`const response = await I.sendGetRequest(readMoreLink);`
replaced with readMoreLink